src/agent/src/sandbox.rs

```diff
@@ -236,14 +236,29 @@ impl Sandbox {
             return Ok(());
         }
 
-        let cpuset = rustjail_cgroups::fs::get_guest_cpuset()?;
+        let guest_cpuset = rustjail_cgroups::fs::get_guest_cpuset()?;
 
         for (_, ctr) in self.containers.iter() {
+            let cpu = ctr
+                .config
+                .spec
+                .as_ref()
+                .unwrap()
+                .linux
+                .as_ref()
+                .unwrap()
+                .resources
+                .as_ref()
+                .unwrap()
+                .cpu
```

Comment:
I wonder if this would not be more readable with something like:
```
.config.spec.as_ref().unwrap()
.linux.as_ref().unwrap()
.resources.as_ref().unwrap()
.cpu
```

The idea being to separate the hierarchy that you are trying to find from the details of how you access it.